CIGIHub/cigionline
# CIGI-730: Implement separate queue for promotion blocks

#### Description of changes
resolves CIGIHub/cigi-tickets#730

- Implemented `Features` app
- Added `Features` admin menu item
- Separated home page promotion blocks section to its own page, and allow blocks to be scheduled
- Clear home page promotion blocks template fragment cache when promotion blocks list page is saved 

To test:
- migrate
- create a features page under Home
- create a Home Page Promotions Page under the features page
- add blocks and save. This should override the promotions blocks set on the home page

#### Pull Request checklist
This checklist needs to be completed before assigning reviewers. If the checklist will not be completed, please comment with the reason.
- [x] Have you reviewed your own code changes?
- [x] Has the issue owner reviewed your changes?
- [x] Have you given the PR a descriptive title?
- [x] Have you described your changes above? Always include screenshots if applicable.
- [x] Have you pushed your latest commit to this branch?

---
#### Code Review checklist
This checklist should be completed if applicable before approving the pull request.
- [x] Have you reviewed the code changes?
- [x] Have you tested the changes on Google Chrome?
- [ ] Have you tested the changes on Safari?
- [ ] Have you tested the changes on Microsoft Edge (non-Chromium)?
- [ ] Have you tested the changes on iOS?
- [ ] Have you tested the changes on Android?
